include/open62541/plugin/securitypolicy.h

```diff
@@ -377,6 +377,27 @@ struct UA_PubSubSecurityPolicy {
 
 #endif
 
+#ifdef UA_ENABLE_PUBSUB_ENCRYPTION_TPM
```

Comment:
Use the existing `UA_PubSubSecurityPolicy`.
If required, make a proposal for forwarding a "context" pointer into a new writerGroupContext.

Reply:
Let me stress this, we will not duplicate interfaces for the TPM.
Reuse the `UA_PubSubSecurityPolicy`.